packages/edit-site/src/components/template-switcher/index.js

```diff
@@ -175,15 +202,6 @@ export default function TemplateSwitcher( {
 					</>
 				) }
 			</DropdownMenu>
-			<AddTemplate
```

Comment:
removing this will definitely break the dirty state entity e2e tests. Not sure what the best path forward is for those e2e tests considering that the UI is likely to continue changing a lot.

Reply:
Can we update them to use the API directly?

Reply:
🤔 Maybe? Like the WP API or the block-editor API? I feel like the e2e test ought to be going through real user actions at the end of the day. Otherwise, the unit tests on `useBlockSync` would be good enough, right?

Reply:
Yeah, I agree. It's just that the UI will probably change a lot with the new designs coming up. Using the WP API, for now, would let us have some sort of safety net, without having to have significant test rewrites as we iterate.

That said, we don't have that many tests right now, I think it makes sense to update them in this PR.